code/modules/language/language_holder.dm

```diff
@@ -296,10 +296,74 @@ Key procs
+							/datum/language/calcic = list(LANGUAGE_ATOM),
+							/datum/language/voltaic = list(LANGUAGE_ATOM))
+
+/datum/language_holder/moth
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/moffic = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/moffic = list(LANGUAGE_ATOM))
+
+/datum/language_holder/skeleton
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/calcic = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/calcic = list(LANGUAGE_ATOM))
+
+/datum/language_holder/ethereal
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/voltaic = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/voltaic = list(LANGUAGE_ATOM))
+
+/datum/language_holder/golem
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/terrum = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/terrum = list(LANGUAGE_ATOM))
+
+/datum/language_holder/fly
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/buzzwords = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/buzzwords = list(LANGUAGE_ATOM))
+
+/datum/language_holder/plant
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/sylvan = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/sylvan = list(LANGUAGE_ATOM))
+
+/datum/language_holder/shadowpeople
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/shadowtongue = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/shadowtongue = list(LANGUAGE_ATOM))
+
+/datum/language_holder/bonegolem
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/terrum = list(LANGUAGE_ATOM),
+								/datum/language/calcic = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/terrum = list(LANGUAGE_ATOM),
+							/datum/language/calcic = list(LANGUAGE_ATOM))
+
+/datum/language_holder/runegolem
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+								/datum/language/terrum = list(LANGUAGE_ATOM),
+								/datum/language/narsie = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
+							/datum/language/terrum = list(LANGUAGE_ATOM),
+							/datum/language/narsie = list(LANGUAGE_ATOM))
```

Comment:
These two should be children of the golem holder, ie /datum/language_holder/golem/bone.
You will still have to set all the languages, but it makes the structure a bit more obvious and allows for changing other stuff for all the golems if that ever becomes a thing.

Reply:
Issues sorted out.